SimPEG/EM/Static/IP/ProblemIP.py

```diff
@@ -47,68 +54,111 @@ def fields(self, m):
             u = self.Ainv * RHS
             Srcs = self.survey.srcList
             self.f[Srcs, self._solutionType] = u
-        return self.f
 
-    def Jvec(self, m, v, f=None):
+    def getJ(self, f=None):
```

Comment:
similar comments as in #650 for the structure of the sensitivity and reducing repeat code